app/models/user.rb

```diff
@@ -1,2 +1,10 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+
+  def recent_post
+    # posts.order (:created_at :desc).limit(3)
```

Comment:
- [ ] Kindly make sure to remove this commented code, so the base code looks clean and easy to navigate